doc/changelog.md

```diff
@@ -1,55 +1,100 @@
-# Release 0.3.2-dev
+# Release 0.3.2
 
 <h3>New features</h3>
 
-<h3>Improvements</h3>
+* The experimental AutoGraph feature now supports Python `while` loops, allowing native Python loops
+  to be captured and compiled with Catalyst.
+  [(#318)](https://github.com/PennyLaneAI/catalyst/pull/318)
 
-* Improve the compiler driver diagnostic output even more. Now it dumps the failing IR as well as
-  prints names of failing passes.
-  [(#349)](https://github.com/PennyLaneAI/catalyst/pull/349)
+  ```python
+  dev = qml.device("lightning.qubit", wires=4)
 
-* Return values of conditional functions no longer need to be of exactly the same type. Catalyst
-  would apply type promotion to branch return values if their types don't match.
-  [(#333)](https://github.com/PennyLaneAI/catalyst/pull/333)
+  @qjit(autograph=True)
+  @qml.qnode(dev)
+  def circuit(n: int, x: float):
+      i = 0
+
+      while i < n:
+          qml.RX(x, wires=i)
+          i += 1
+
+      return qml.expval(qml.PauliZ(0))
+  ```
+
+  ```pycon
+  >>> circuit(4, 0.32)
+  array(0.94923542)
+  ```
+
+  This feature extends the existing AutoGraph support for Python `for` loops and if` statements
+  introduced in v0.3. Note that TensorFlow must be installed for AutoGraph support.
+
+  For more details, please see the
+  [AutoGraph guide](https://docs.pennylane.ai/projects/catalyst/en/stable/dev/autograph.html).
+
+* In addition to loops and conditional branches, AutoGraph now supports native Python `and`, `or`
```

Comment:
We need to add these new supports to the AutoGaraph guide page

Reply:
I didn't review these PRs, otherwise I'd recommend it there

Reply:
Doing that now :)